docs/Getting Started/Installation/index.md

```diff
@@ -4,11 +4,9 @@ The following must be installed before installation of the Policy Server can beg
 | Project | Version |
 |---------|---------|
 | `Postgres` | 9.6+ |
-| `Node.js` | 4.0.0 - 12.22.0 |
+| `Node.js` | 8.12.0+ |
 | `NPM` | 3.0.0+ |
 
-Note the maximum Node version. **The policy server will not work on Node versions 13 or higher.**
```

Comment:
Since we don't say it anywhere else and OEMs may not upgrade immediately, we should still have something about policy server 2.0's supported node version range.